cf

```diff
@@ -557,6 +555,9 @@ if(defined($GENOME) && defined($BOWTIE_PATHS{$GENOME})){
 if(defined($GENOME) && defined($BOWTIE2_PATHS{$GENOME})){
 	$runfile .= "\@bowtie2_path\t$BOWTIE2_PATHS{$GENOME}\n";
 }
+elsif (defined($GENOME)) {
```

Comment:
Won't this always fire warnings, even if the current run has nothing to do with bowtie2? Doesn't CF already issue a warning and exit later on if it needs a bowtie2 path and doesn't have one?